src/rapids_dependency_file_generator/cli.py

```diff
@@ -35,10 +37,17 @@ def validate_args(argv):
 
     codependent_args = parser.add_argument_group("optional, but codependent")
     codependent_args.add_argument(
-        "--file_key",
+        "-f",
+        "--file-key",
         help="The file key from `dependencies.yaml` to generate",
     )
     codependent_args.add_argument(
+        "--file_key",
+        dest="file_key_deprecated",
+        help="Deprecated alias for --file-key",
```

Comment:
Do you want to spell this deprecation the same way as below?
```suggestion
        help="DEPRECATED: Use --file-key instead.",
```